openmc_dagmc_wrapper/tallies/mesh_tallies/mesh_tally.py

```diff
@@ -0,0 +1,11 @@
+import openmc
+
+from openmc_dagmc_wrapper import Tally
+
+
+class MeshTally(Tally):
+    def __init__(self, tally_type, mesh, **kwargs):
+        super().__init__(tally_type, **kwargs)
+        self.mesh = mesh
+        self.name = self.tally_type + "_on_mesh"
+        self.filters.append(openmc.MeshFilter(self.mesh))
```

Comment:
Nothing more convenient here, more an abstraction since `name` has a sort of default value.
It can be changed, but then it defeats the purpose